### PR TITLE
Queue content notifications tokens

### PR DESCRIPTION
**Pull request by markrichardsg**

### CODE REVIEW

File: notificationworkerlambda/src/test/scala/com/gu/notifications/worker/WorkerRequestHandlerSpec.scala [hunk 1]
```diff
@@ -28,29 +28,46 @@ import scala.collection.JavaConverters._
 class WorkerRequestHandlerSpec extends Specification with Matchers {
 
   "the WorkerRequestHandler" should {
-    "Send one notification" in new WRHSScope {
-      workerRequestHandler.handleRequest(sqsEventShardNotification, null)
+    "Send one content notification" in new WRHSScope {
+      workerRequestHandler.handleRequest(sqsEventShardNotification(breakingNewsNotification), null)
```
Comment (NathanielBennett) [inline notificationworkerlambda/src/test/scala/com/gu/notifications/worker/WorkerRequestHandlerSpec.scala]: is a breaking news notification a content notification?

File: notificationworkerlambda/src/test/scala/com/gu/notifications/worker/WorkerRequestHandlerSpec.scala [hunk 2]
```diff
@@ -84,13 +103,25 @@ class WorkerRequestHandlerSpec extends Specification with Matchers {
       importance = Major,
       topic = List(Topic(Breaking, "uk"), Topic(Breaking, "us"), Topic(Breaking, "au"), Topic(Breaking, "international"))
     )
-    val shardedNotification = ShardedNotification(
-      notification = notification,
-      range = ShardRange(0, 1),
-      platform = Some(Android)
+    val contentNNotification = BreakingNewsNotification(
```
Comment (NathanielBennett) [inline notificationworkerlambda/src/test/scala/com/gu/notifications/worker/WorkerRequestHandlerSpec.scala]: typo in  val name?

File: notificationworkerlambda/src/test/scala/com/gu/notifications/worker/WorkerRequestHandlerSpec.scala (hunk 1 above)
Comment (markrichardsg) [reply notificationworkerlambda/src/test/scala/com/gu/notifications/worker/WorkerRequestHandlerSpec.scala]: 🤦‍♂️ good spot

File: notificationworkerlambda/src/test/scala/com/gu/notifications/worker/WorkerRequestHandlerSpec.scala (hunk 2 above)
Comment (markrichardsg) [reply notificationworkerlambda/src/test/scala/com/gu/notifications/worker/WorkerRequestHandlerSpec.scala]: Yes, fixing

**Comment by alexduf** [comment]
Let us know if you need more eyes 👀 when merging.

**Pull request merged**